site/content/troubleshooting-guide.md

```diff
@@ -1 +1,122 @@
-### TODO
\ No newline at end of file
+
+# Troubleshooting Guide
+
+This guide outlines some failure scenarios and their possible solutions.
+
+## Node.js not installed
+AWS.Deploy.Tools relies on AWS CDK to provision resources for the user's cloud application. AWS CDK requires Node.js to be installed in the user's machine.
```

Comment:
Can you spell out CDK the first time you use it? And link to their docs?

They also have a nice FAQ entry explaining the Node dependency that we can link to - https://aws.amazon.com/cdk/faqs/

Reply:
Expanded CDK for first time usage. There is no hyperlink to that specific FAQ entry so I haven't included it.

Reply:
1. Could you link to https://aws.amazon.com/cdk/ from the first CDK reference?
2. I still think it'd be nice to link to the FAQ page even if there isn't a specific anchor, something like _"See the [CDK's FAQs](https://aws.amazon.com/cdk/faqs/) for more information about how it uses Node.js"_

Reply:
Linked the CDK documentation for the first CDK reference. Also included the link to the CDK FAQ.